### ISSUE TITLE
app crash

### ISSUE THREAD

**Issue by SzewczykAdam** [nextcloud/android#6894]
************ CAUSE OF ERROR ************

android.database.sqlite.SQLiteBlobTooBigException: Row too big to fit into CursorWindow requiredPos=2848, totalRows=86
	at android.database.sqlite.SQLiteConnection.nativeExecuteForCursorWindow(Native Method)
	at android.database.sqlite.SQLiteConnection.executeForCursorWindow(SQLiteConnection.java:942)
	at android.database.sqlite.SQLiteSession.executeForCursorWindow(SQLiteSession.java:838)
	at android.database.sqlite.SQLiteQuery.fillWindow(SQLiteQuery.java:62)
	at android.database.sqlite.SQLiteCursor.fillWindow(SQLiteCursor.java:161)
	at android.database.sqlite.SQLiteCursor.onMove(SQLiteCursor.java:131)
	at android.database.AbstractCursor.moveToPosition(AbstractCursor.java:249)
	at android.database.AbstractCursor.moveToNext(AbstractCursor.java:281)
	at com.owncloud.android.providers.FileContentProvider.deleteDirectory(FileContentProvider.java:190)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:134)
	at com.owncloud.android.providers.FileContentProvider.deleteDirectory(FileContentProvider.java:184)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:134)
	at com.owncloud.android.providers.FileContentProvider.delete(FileContentProvider.java:114)
	at android.content.ContentProviderOperation.applyInternal(ContentProviderOperation.java:332)
	at android.content.ContentProviderOperation.apply(ContentProviderOperation.java:309)
	at com.owncloud.android.providers.FileContentProvider.applyBatch(FileContentProvider.java:672)
	at android.content.ContentProvider.applyBatch(ContentProvider.java:2117)
	at android.content.ContentProvider$Transport.applyBatch(ContentProvider.java:371)
	at android.content.ContentProviderClient.applyBatch(ContentProviderClient.java:532)
	at android.content.ContentProviderClient.applyBatch(ContentProviderClient.java:520)
	at android.content.ContentResolver.applyBatch(ContentResolver.java:1875)
	at com.owncloud.android.datamodel.FileDataStorageManager.saveFolder(FileDataStorageManager.java:435)
	at com.owncloud.android.operations.RefreshFolderOperation.synchronizeData(RefreshFolderOperation.java:490)
	at com.owncloud.android.operations.RefreshFolderOperation.fetchAndSyncRemoteFolder(RefreshFolderOperation.java:372)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:231)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:360)
	at java.lang.Thread.run(Thread.java:919)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30130090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: Xiaomi
Device: perseus
Model: Mi MIX 3
Id: QQ2A.200405.005
Product: perseus

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 448bbff48a

**Comment by Github-Berny** [comment]
My app also crashes. After entering the server adddress I get a white screen for milliseconds and than it turns to a nextcloud-blue screen. That's it! I'm not asked for login credentials. The same behavior with the latest app version but also with 13.2.0 and 13.2.1.
Reason may be: Due to the upcomming Nextcloud version 20 which requires PHP 7.4 I set up my Nextcloud computer completely new using Ubuntu 20.04… The only difference in my Nextcloud configuration to the old one is a new ssl configuration: formerly only rsa certificate, now ecc plus rsa certificates.
The conclusion for me seem to be that the nextcloud app can’t handle the new certificate configuration. But without any error messages we will never know, because I don't get an nginx error.log or nextcloud.log issue for that!
I have attached my nextcloud config files, hopefully it can help....
[brxsronok8zu6a1o.dedyn.io.txt](https://github.com/nextcloud/android/files/5176344/brxsronok8zu6a1o.dedyn.io.txt)
[brxsronok8zu6a1o.dedyn.io_letsencrypt.txt](https://github.com/nextcloud/android/files/5176345/brxsronok8zu6a1o.dedyn.io_letsencrypt.txt)
[brxsronok8zu6a1o.dedyn.io_nextcloud.txt](https://github.com/nextcloud/android/files/5176346/brxsronok8zu6a1o.dedyn.io_nextcloud.txt)
